snuba/query/allocation_policies/bytes_scanned_rejecting_policy.py

```diff
@@ -257,7 +257,17 @@ def _get_quota_allowance(
                 suggestion=SUGGESTION,
             )
 
-        if granted_quota.granted < throttle_threshold:
+        (
+            customer_tenant_key,
+            customer_tenant_value,
+        ) = self._get_customer_tenant_key_and_value(tenant_ids)
+        if (
+            granted_quota.granted
+            < self.__get_scan_limit(
+                customer_tenant_key, customer_tenant_value, referrer
+            )
+            - throttle_threshold
+        ):
```

Comment:
Add a comment to explain what you're doing here so someone can easily understand it